contracts/upgradeable_contracts/BasicOmnibridge.sol

```diff
@@ -37,6 +37,20 @@ abstract contract BasicOmnibridge is
     using SafeERC20 for IERC677;
     using SafeMath for uint256;
 
+    // Workaround for storing variable up-to-32 bytes suffix
+    uint256 private immutable SUFFIX_SIZE;
+    bytes32 private immutable SUFFIX;
```

Comment:
Don't you think that nothing stops us to make this parameter configurable on runtime not only during deployment?

Reply:
I don't think it might be useful. So, since it will increase the bytecode size and average gas usage, I would stick to the immutable constants for that.

Reply:
agree